src/Admin/AdminConsole/Models/CreateMultiOrganizationEnterpriseProviderModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.AdminConsole.Entities.Provider;
+using Bit.Core.Billing.Enums;
+using Bit.SharedWeb.Utilities;
+
+namespace Bit.Admin.AdminConsole.Models;
+
+public class CreateMultiOrganizationEnterpriseProviderModel : IValidatableObject
+{
+    [Display(Name = "Owner Email")]
+    public string OwnerEmail { get; set; }
+
+    [Display(Name = "Enterprise Seat Minimum")]
+    public int EnterpriseSeatMinimum { get; set; }
+
+    [Display(Name = "Plan")]
+    [Required]
```

Comment:
This improves the error message returned when no plan was selected. Otherwise you get: "The value '' is invalid". Now it returns a proper error message.